crypto-ffi/src/error/wasm.rs

```diff
@@ -65,4 +89,30 @@ impl CoreCryptoError {
     pub(crate) fn ad_hoc(err: impl ToString) -> Self {
         Self(InternalError::ad_hoc(err))
     }
+
+    fn get_type_and_context(&self) -> (&str, wasm_bindgen::JsValue) {
+        match &self.0 {
+            e @ super::core_crypto::CoreCryptoError::Mls { mls_error } => (e.as_ref(), mls_error.get_context()),
+            e @ super::core_crypto::CoreCryptoError::Proteus {
+                exception: proteus_error,
+            } => (e.as_ref(), proteus_error.get_context()),
+            e @ super::core_crypto::CoreCryptoError::E2ei { e2ei_error } => {
+                (e.as_ref(), JsValue::new_with_property("e2ei_error", e2ei_error))
```

Comment:
What's the thinking here, sometimes we convert the error into a string and sometimes we pass it as is?

Reply:
e2ei_errror is a string in this case, so now conversion needed here